### ISSUE TITLE
Preview mode exploding model

### ISSUE THREAD

**Issue by docksteaderluke** [Ultimaker/Cura#10057]
### Application Version

4.9.1 and 4.10

### Platform

macOS Big Sur v11.4

### Printer

Creality Ender 5 Pro

### Reproduction steps

1. Upgrade to 4.10

### Actual results

Everything now explodes in Preview mode even after downgrading to 4.9.1.
This issue did not occur prior to update.
<img width="996" alt="Screen Shot 2021-06-30 at 12 15 09" src="https://user-images.githubusercontent.com/1275275/123989028-e2ea5400-d99e-11eb-91d0-62965c48edb7.png">

### Expected results

Model shouldn't explode.

### Checklist of files to include

- [ ] Log file
- [ ] Project file

### Additional information & file uploads

Changing slicer settings doesn't change anything.
This happens to _all_ models now. I've tested with both new models and those that I previously confirmed as working before the update.
No crashing occurs.

**Comment by docksteaderluke** [comment]
Hi @fvrmr,

No, I do not! Here are my machine's specs.

<img width="240" alt="Screen Shot 2021-07-02 at 20 04 23" src="https://user-images.githubusercontent.com/1275275/124335456-b4bc6e00-db70-11eb-9a3e-2b2b890b8383.png">

**Comment by Ghostkeeper** [comment]
You also posted this on Reddit. I recognise the picture. Here is the post: https://www.reddit.com/r/Cura/comments/oazfff/just_upgraded_to_cura_410_and_my_preview_mode_is/

I have to agree with ahoeben there that this is most likely a graphics driver issue. Nothing really changed between 4.9 and 4.10 with how we render layer view (other than the addition of another colour scheme which you're not using here). You reported there that you also upgraded MacOS itself at the same time, which seems like a more likely culprit.

Could you disambiguate by seeing if the same issue also occurred in Cura 4.9 while using your new MacOS version?

So far none of our developers have been able to reproduce this issue. We haven't seen it happen yet.

**Comment by docksteaderluke** [comment]
Yes — I had originally posted it on Reddit then thought this might be a more appropriate place.

I've tried a number of things over the last few days and rebooted my MacBook a few times and, strangely enough, the last time I rebooted it seems to have fixed the issue (not sure why it didn't fix it the first couple of restarts) 🤷.

At this point I can no longer reproduce the bug either so I think it's safe to close this issue. I'll report back if the issue pops up again.

Thanks for your help!
